frontend/src/components/eventPage/EventPage.jsx

```diff
@@ -9,30 +9,30 @@ const EventPage = () => {
     const current_eventId = current_link.split('/')[2]
     const [event, setEvent] = useState()
     const [comments, setComments] = useState()
-    let [changeInEvent, setchangeInEvent] = useState(true)
     const [isLoading, setIsLoading] = useState(true)
 
     useEffect(() => {
-        if (changeInEvent) {
-            setIsLoading(true)
-            fetchEvent()
-            fetchComments()
-        }
-        setIsLoading(false)
-        setchangeInEvent(false)
+        fetchEvent()
+        fetchComments()
     }, [event])
```

Comment:
https://www.w3schools.com/react/react_useeffect.asp
With your current logic, event and comments are re-fetched every time event state has changed. Is that what you want to achieve? Or do you want to re-fetch them every render?

Reply:
Every render.
Thank you, I didn't know you could do this.

Reply:
Would it be better to fetch it only in the first render?